modules/components/org.wso2.carbon.appfactory.provisioning.runtime/src/main/java/org/wso2/carbon/appfactory/provisioning/runtime/RuntimeProvisioningService.java

```diff
@@ -23,45 +23,177 @@
 
 public interface RuntimeProvisioningService {
```

Comment:
All interface methods are by default "public and abstract". Therefore, explicit declarations of access modifiers upon method signatures is redundant.